tests/unit/adapters/stripe-account-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('adapter:stripe-account', 'Unit | Adapter | stripe account', {
+  // Specify the other units that are required for this test.
+  // needs: ['serializer:foo']
+});
+
+// Replace this with your real tests.
+test('it exists', function(assert) {
```

Comment:
I don't know if that can happen at a unit test level as we have to test the request itself. Even then, I may argue against it since it seems like we are testing `Adapter` class.